installer/manifests/common/templates/_images.tpl

```diff
@@ -0,0 +1,39 @@
+{{/*Return the proper serivce image name*/}}
+{{- define "service.image" -}}
+{{/*{{- include "common.images.image" ( dict "imageRoot" .Values.helmservice.image "global" .Values.global.keptn "defaultTag" .Chart.appVersion) -}}*/}}
+{{- $registryName := "" -}}
+{{- $repositoryName := .imageRoot.repository -}}
+{{- $tag := include "service.tag" (dict "imageRoot" .imageRoot "global" .global "defaultTag" .defaultTag) -}}
+{{- if .global }}
+    {{- if .global.registry }}
+     {{- $registryName = .global.registry -}}
+    {{- end -}}
+{{- end -}}
+{{- if .imageRoot.registry -}}
+  {{- $registryName = .imageRoot.registry -}}
+{{- end -}}
+{{- if $registryName }}
+{{- printf "%s/%s:%s" $registryName $repositoryName $tag -}}
+{{- else -}}
+{{- printf "%s:%s" $repositoryName $tag -}}
+{{- end -}}
+{{- end -}}
+
+{{/*Return the proper serivce image tag*/}}
+{{- define "service.tag" -}}
+{{/*{{- include "common.images.image" ( dict "imageRoot" .Values.helmservice.image "global" .Values.global.keptn "defaultTag" .Chart.appVersion) -}}*/}}
```

Comment:
If my comment above about the naming convention here make sense, this would be renamed to `common.images.tag`
```suggestion
{{/*
Return the proper service image tag
{{- include "common.images.tag" ( dict "imageRoot" .Values.yourService.image "global" .Values.global.keptn "defaultTag" .Chart.appVersion) -}}
*/}}
{{- define "common.images.tag" -}}
```